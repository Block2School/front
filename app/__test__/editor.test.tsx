import renderer from 'react-test-renderer'
import React from 'react'
import MyEditor from '../components/editor/editor'
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import { shallow } from 'enzyme';
import { render, screen } from '@testing-library/react';


describe('<MyEditor/>', () => {
  test('should render correctly', () => {
    const theme = 'vs-dark';
    const lang = 'javascript';
    shallow(<MyEditor theme={theme} lang={lang} onChange={undefined} defaultValue={undefined} />)
  });
})