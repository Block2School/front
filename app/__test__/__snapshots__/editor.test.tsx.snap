// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MyEditor/> match snapshot 1`] = `ShallowWrapper {}`;
